### PR TITLE
Feature/issue 27 global config deleted at fieldname

### PR DESCRIPTION
**Pull request by ak47**

### CODE REVIEW

File: spec/mongoid/paranoia_spec.rb [hunk 1]
```diff
@@ -2,6 +2,28 @@
...
+      end
+
+      class ParanoidConfigured
+        include Mongoid::Document
+        include Mongoid::Paranoia
+      end
+    end
+
+    it 'allows custom setting of the paranoid_field' do
+      paranoid_configurd = ParanoidConfigured.new
```
Comment (simi) [inline spec/mongoid/paranoia_spec.rb]: Is this typo `paranoid_configurd` => `paranoid_configured` ?

**Comment by simi** [comment]
This is really nice! I found a typo probably. Please check my comment. Also is there any chance to provide this as one commit (rebase)?

:heart: Thanks for PR! :heart:

**Comment by ak47** [comment]
fixed var name typo, and sorted out the rebase

thank you!

File: lib/mongoid/paranoia.rb [hunk 2]
```diff
@@ -18,8 +19,31 @@ module Paranoia
...
 
+    class << self
+      attr_accessor :configuration
+    end
+
+    def self.configuration
+      @configuration ||= Configuration.new
+    end
+
+    def self.reset
```
Comment (simi) [inline lib/mongoid/paranoia.rb]: @ak47 one last question. Is there chance to test reset also?

Comment (ak47) [inline lib/mongoid/paranoia.rb]: Added spec for Mongoid::Paranoia.reset, it's a little odd due to how the configuration gets set when included into the defined class - you'll see in the spec

Also cleaned up the configuration_spec so it's more readable.

**Comment by johnnyshields** [comment]
:+1:

**Comment by simi** [comment]
Thanks for confirmation @johnnyshields.

**Pull request merged**